azure-mgmt-cosmosdb/src/main/java/com/microsoft/azure/management/cosmosdb/CosmosDBAccount.java

```diff
@@ -133,6 +134,50 @@
      */
     boolean keyBasedMetadataWriteAccessDisabled();
 
+    /**
+     * @return all private link resources in the account.
+     */
+    Observable<List<PrivateLinkResource>> listPrivateLinkResourcesAsync();
+
+    /**
+     * @return all private link resources in the account.
+     */
+    List<PrivateLinkResource> listPrivateLinkResources();
+
+    /**
+     * @param groupName group name of private link resource
```

Comment:
groupName or groupId or privateLinkName?

Reply:
the api is called groupName https://github.com/Azure/azure-rest-api-specs/blob/de100e16eabbd5d41551cba8b1d8cd97f3701bf9/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/privateLinkResources.json#L76

Reply:
but the sample response is like groupId https://github.com/Azure/azure-rest-api-specs/blob/de100e16eabbd5d41551cba8b1d8cd97f3701bf9/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2019-08-01-preview/examples/CosmosDBPrivateLinkResourceGet.json#L16